Translations/uk.lproj/Localizable.strings

```diff
@@ -212,21 +212,21 @@ Apple needs to have MAS translated in this language so this strings can be used
 The language used in MAS/AppleID will be used in Keka's tip strings and may not match Keka's localization
 Note that the short string is limited to 30 characters and the long string is limited to 45 characters.
 */
-"Monthly sip of coffee" = "Monthly sip of coffee";
-"Gift Keka a sip of coffee each month" = "Gift Keka a sip of coffee each month";
-"Monthly coffee" = "Monthly coffee";
-"Gift Keka a coffee each month" = "Gift Keka a coffee each month";
-"Monthly pair of coffees" = "Monthly pair of coffees";
-"Gift Keka a couple coffees each month" = "Gift Keka a couple coffees each month";
-"Monthly breakfast" = "Monthly breakfast";
-"Gift Keka a breakfast each month" = "Gift Keka a breakfast each month";
-"Tip a coffee" = "Tip a coffee";
-"Tip Keka a coffee so it can stay awake" = "Tip Keka a coffee so it can stay awake";
-"Tip a breakfast" = "Tip a breakfast";
-"Tip Keka a breakfast so it can start fresh" = "Tip Keka a breakfast so it can start fresh";
-"Tip a lunch" = "Tip a lunch";
-"Tip Keka a lunch and feed it" = "Tip Keka a lunch and feed it";
-"Tip a dinner" = "Tip a dinner";
-"Tip Keka a dinner so it keeps rolling" = "Tip Keka a dinner so it keeps rolling";
+"Monthly sip of coffee" = "Щомісячний ковток кави";
+"Gift Keka a sip of coffee each month" = "Подаруйте Keka щомісячний ковток кави";
+"Monthly coffee" = "Щомісячна кава";
+"Gift Keka a coffee each month" = "Подаруйте Keka щомісячну каву";
+"Monthly pair of coffees" = "Щомісячна пара кави";
+"Gift Keka a couple coffees each month" = "Подаруйте Keka щомісячну пару кав";
+"Monthly breakfast" = "Щомісячний сніданок";
+"Gift Keka a breakfast each month" = "Подаруйте Keka щомісячний сніданок";
+"Tip a coffee" = "Чайові на каву";
+"Tip Keka a coffee so it can stay awake" = "Дайте Keka кави, щоб вона могла не спати";
+"Tip a breakfast" = "Чайові на сніданок";
+"Tip Keka a breakfast so it can start fresh" = "Дайте Keka сніданок, щоб почати все з чистого аркуша";
```

Comment:
@likhner this one can't be used because exceeds 45 characters

Reply:
I'll ask @hugonote to fix it then (because he does Ukrainian translation). He was just double-checking the current translation. He will also make a new PR as well.

Maybe you should mention in the README about the 45-character limit?

Reply:
Ah thanks @likhner! I've added some comments about the limit in the file itself but it's common (and understandable) to skip that comments...

https://github.com/aonez/Keka/blob/2fc95ddf979455e6f86878635358012e37aec57c/Translations/uk.lproj/Localizable.strings#L228-L253